packages/notification-center/package.json

```diff
@@ -62,6 +63,7 @@
     "lodash.merge": "^4.6.2",
     "react-infinite-scroll-component": "^6.0.0",
     "react-query": "^3.34.19",
+    "react-to-webcomponent": "2.0.0-alpha.0",
```

Comment:
we don't need this anymore

Reply:
Not sure if this comment is in the right place because you are adding rather than removing.

Reply:
that one was for me to not forget to remove it ;)